target/linux/ramips/mt7621/target.mk

```diff
@@ -6,6 +6,15 @@ SUBTARGET:=mt7621
 BOARDNAME:=MT7621 based boards
 FEATURES+=nand ramdisk rtc usb minor
 CPU_TYPE:=24kc
+# Build kernel vmlinuz target.
+# For include/kernel-defaults.mk: Kernel/CompileImage
+KERNELNAME:=vmlinuz
+# Use vmlinuz. For include/kernel-defaults.mk: Kernel/CopyImage
+KERNEL_IMAGES:=vmlinuz
```

Comment:
I am not sure if this is the correct place for these, or if they can be set per-device?
Only one of these is needed on mips, but they are left for the example.
[vmlinuz is added to the kernel all target with ZBOOT support](https://elixir.bootlin.com/linux/v5.4.66/source/arch/mips/Makefile#L83), and because "vmlinuz" is used as KERNEL_NAME, it would get copied out anyway. Arm uses vmlinux for the compressed name, so would need something here?

Where used:
- If KERNEL_IMAGES, copy it out. Otherwise, whatever is in KERNELNAME, which is not vmlinux or dtbs, copy them out.
  https://github.com/openwrt/openwrt/blob/83fcc0e6ae4d370de154e6bf23e2fbf367f25946/include/kernel-defaults.mk#L127
  ```
  $(foreach k, \
  		$(if $(KERNEL_IMAGES),$(KERNEL_IMAGES),$(filter-out vmlinux dtbs,$(KERNELNAME))), \
  		$(CP) $(LINUX_DIR)/arch/$(LINUX_KARCH)/boot/$(IMAGES_DIR)/$(k) $(KERNEL_BUILD_DIR)/$(k)$(1); \
  ) \
  ```

- If KERNELNAME is set, use it as the linux target, otherwise, use all.
  https://github.com/openwrt/openwrt/blob/83fcc0e6ae4d370de154e6bf23e2fbf367f25946/include/kernel-defaults.mk#L139
  ```
  +$(KERNEL_MAKE) $(if $(KERNELNAME),$(KERNELNAME),all) modules
  ```

Reply:
mediatek/mt7623 is also using multiple images provided by kernel. You could check how things are done there.